jenkins/catapult/configurations/gcc-code-coverage.yaml

```diff
@@ -1,3 +1,3 @@
-compiler: ../compilers/gcc-10.yaml
+compiler: ../compilers/gcc-11.yaml
```

Comment:
1. should this be gcc latest (12) ?
2. should we have a clang-latest.yaml gcc-latest.yaml to minimize future changes when bumping versions ?

Reply:
1. Yes, can move it to 12.  All the daily tests should run with the latest compiler?  Weekly tests are the current - 1 compiler version?
2. Good idea, I think that would help

Reply:
>  the daily tests should run with the latest compiler? Weekly tests are the current - 1 compiler version

i think that is fine, @gimre-xymcity